tests/components/kodi/test_init.py

```diff
@@ -0,0 +1,19 @@
+"""Test the Kodi integration init."""
+from homeassistant.components.kodi.const import DOMAIN
+from homeassistant.config_entries import ENTRY_STATE_LOADED, ENTRY_STATE_NOT_LOADED
+
+from . import init_integration
```

Comment:
Where is this defined?

```
tests/components/kodi/test_init.py:5: in <module>
    from . import init_integration
E   ImportError: cannot import name 'init_integration' from 'tests.components.kodi' (/Users/onfreund/dev/personal/home-assistant/core/tests/components/kodi/__init__.py)
```

Reply:
working through github took a second to get the other file updated

Reply:
👍